### DISCUSSION TITLE
What's the best Chakra-ui replacement for react-multi-carousel?

### DISCUSSION THREAD

**Discussion by wadewadewadewadewadewade** [chakra-ui/chakra-ui#4361] [Q&A]
Greetings!

I'm looking for a good, accessible, Chakra-ui-based Carousel to replace  the NPM [react-multi-carousel](https://www.npmjs.com/package/react-multi-carousel) package we currently use with styled-components and semantic-ui.

I didn't see one as part of the core Chakrau-ui library yet, but I did see this issues thread where the user kripod seems to have built a good Carousel candidate that it looks like I would have to modify a bit: https://github.com/chakra-ui/chakra-ui/issues/200

But I'd love to hear if there are other Chakra-ui Carousels out there that I was unable to find, or if building one like `react-multi-carousel` is on the development roadmap somewhere. What's out there with respect to Chakra-ui carousels - what do you use?

Thank you!

**Comment by anubra266** [comment]
https://choc-ui.tech/docs/elements/carousels
